BrainPortal/app/views/tool_configs/_form_fields.html.erb

```diff
@@ -22,172 +21,187 @@
 #
 -%>
 
-<div class="groupentry">
 
-  <%= hidden_field_tag "tool_config[tool_id]",     @tool_config.tool_id %>
-  <%= hidden_field_tag "tool_config[bourreau_id]", @tool_config.bourreau_id %>
 
+<%= error_messages_for @tool_config %>
```

Comment:
Show more attributes (uneditable) like date fields

version_name shoudl be first

boxes too wide

too many  /divs